jib-maven-plugin/README.md

```diff
@@ -362,20 +362,27 @@ In this configuration, the image:
 
 You can add arbitrary, non-classpath files to the image by placing them in a `src/main/jib` directory. This will copy all files within the `jib` folder to the image's root directory, maintaining the same structure (e.g. if you have a text file at `src/main/jib/dir/hello.txt`, then your image will contain `/dir/hello.txt` after being built with Jib).
 
-You can configure a different directory by using the `<extraDirectory>` parameter in your `pom.xml`:
+You can configure different directories by using the `<extraDirectories>` parameter in your `pom.xml`:
 ```xml
 <configuration>
-  <!-- Copies files from 'src/main/custom-extra-dir' instead of 'src/main/jib' -->
-  <extraDirectory>${project.basedir}/src/main/custom-extra-dir</extraDirectory>
+  <!-- Copies files from 'src/main/custom-extra-dir' and '/home/user/jib-extras' instead of 'src/main/jib' -->
+  <extraDirectories>
+    <paths>
+      <path>src/main/custom-extra-dir</path>
+      <path>/home/user/jib-extras</path>
+    </paths>
+  </extraDirectories>
 </configuration>
 ```
 
-Alternatively, the `<extraDirectory>` parameter can be used as an object to set a custom extra directory, as well as the extra files' permissions on the container:
+Alternatively, the `<extraDirectories>` parameter can be used as an object to set custom extra directories, as well as the extra files' permissions on the container:
 
 ```xml
 <configuration>
-  <extraDirectory>
-    <path>${project.basedir}/src/main/custom-extra-dir</path> <!-- Copies files from 'src/main/custom-extra-dir' -->
+  <extraDirectories>
+    <paths>
+      <path>src/main/custom-extra-dir</path> <!-- Copies files from 'src/main/custom-extra-dir' -->
```

Comment:
I think in Maven you can skip the inner tags for a single element list, i.e. instead of
```
<paths>
  <path>src/main/custom-extra-dir</path>
</paths>
```
you can just write
```
<paths>src/main/custom-extra-dir</paths>
```

Which might look simpler in this case.

Reply:
Sounds good, as we have the multiple directory case.